engines/native/sandbox.go

```diff
@@ -30,6 +30,25 @@ type sandbox struct {
 	wg         atomics.WaitGroup
 }
 
+// white list of environment variables that must
+// be passed to the child process
+var environmentWhitelist = []string{
+	"PATH",
+	"TMPDIR",
+	"PWD",
```

Comment:
what is this, and it is good to pass down, if we're creating a sub-user...?

in docker images we ususally set:
```
# Set variable normally configured at login, by the shells parent process, these
# are taken from GNU su manual
ENV           HOME          /home/worker
ENV           SHELL         /bin/bash
ENV           USER          worker # username
ENV           LOGNAME       worker # username
ENV           HOSTNAME      taskcluster-worker
ENV           LANG          en_US.UTF-8
ENV           LC_ALL        en_US.UTF-8
```
based on GNU su manual.